examples/full-elastic-stack/k8s/kube-audit-rest.yaml

```diff
@@ -0,0 +1,143 @@
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vector-config
+data:
+  vector.yaml: |
+    sources:
+      audit-files:
+        type: file
+        include:
+          - /tmp/*
+    transforms:
```

Comment:
Probably I would add some limitation to how the json is parsed, as it may easily happened that a plain json read of the message may encounter incompatible fields (For example a field being a single object and then an array of object in another k8s api)

Reply:
Sounds reasonable, mind linking me to an example of this with vector?

Reply:
Also, the point of this is to be as simple to read as possible. How simply can we make that change?